src/main/java/org/example/PhoneNumberValidator.java

```diff
@@ -0,0 +1,19 @@
+package org.example;
+
+import java.util.regex.Pattern;
+import java.util.regex.Matcher;
+
+public class PhoneNumberValidator {
+
+    String phoneNumberRegex = "^(\\+90|0)?(5[0-9]\\d{8}|2\\d{9})$";
```

Comment:
Why `^(\\+90|0)?(5[0-9]\\d{8}|2\\d{9})$` instead of `^(\\+90|0)?(5\\d{9}|2\\d{9})$`